### DISCUSSION TITLE
请提供一份csghub的接口文档

### DISCUSSION THREAD

**Discussion by miss-uny** [OpenCSGs/csghub#599] [Q&A] [answered]
hi，你好，没有在csghub的git代码中找到接口文档，你帮忙提供一下

**Comment by samchen8008** [comment]
https://hub-stg.opencsg.com/api/v1/swagger/index.html

**Comment by miss-uny** [comment, marked as answer]
https://hub.opencsg-stg.com/api/v1/swagger/index.html
